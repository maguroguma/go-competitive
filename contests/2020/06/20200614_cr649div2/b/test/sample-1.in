2
3
3 2 1
4
1 3 4 2
