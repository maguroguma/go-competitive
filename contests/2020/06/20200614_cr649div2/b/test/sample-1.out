2
3 1
3
1 4 2
